ocs_ci/utility/utils.py

```diff
@@ -2574,3 +2575,24 @@ def check_for_rhcos_images(url):
     """
     r = requests.head(url)
     return r.status_code == requests.codes.ok
+
+
+def download_file_from_git_repo(git_repo_url, path_to_file_in_git, filename):
+    """
+    Download a file from a specified git repository
+
+    Args:
+        git_repo_url (str): The git repository url
+        path_to_file_in_git (str): Path to the file to download
+            in git repository
+        filename (str): Name of the file to write the download to
+
+    """
+    log.debug(
+        f"Download file '{path_to_file_in_git}' from "
+        f"git repository {git_repo_url} to local file '{filename}'."
+    )
+    t = mkdtemp()
```

Comment:
can you take a meaningful variable instead of single letter "t"

Reply:
Yes ofc